website/docs/r/backup_policy_vm.markdown

```diff
@@ -99,7 +99,7 @@ The `backup` block supports:
 
 The `retention_daily` block supports:
 
-* `count` - (Required) The number of daily backups to keep. Must be between `1` and `9999`
+* `count` - (Required) The number of daily backups to keep. Must be between `1` and `9999`. This can no longer be created or updated with a value less than `7`.
```

Comment:
Only older users should experience this issue so we can change the documentation to reflect that we only support `7` to `9999` so we don't confuse newer users trying to set this up

```suggestion
* `count` - (Required) The number of daily backups to keep. Must be between `7` and `9999`.
```